fastlane/Fastfile

```diff
@@ -0,0 +1,41 @@
+lane :beta do
+  version = updateVersion("beta")
+  update("beta", version)
+end
+
+lane :release do
+  version = get_version_number(target: "HyperwalletSDK")
+  commit = last_git_commit
+  message = commit[:message]
+  add_git_tag(tag: "#{version}", message: "#{message}")
+  set_github_release(tag_name: "#{version}", description: "#{message}")
+  push_to_git_remote
+  pod_push(allow_warnings: true, skip_tests: true)
+end
+
+def updateVersion(type)
+  version = get_version_number(target: "HyperwalletSDK")
```

Comment:
You can use '**last_git_tag**' instead of '**get_version_number(target: "HyperwalletSDK")**'.